### DISCUSSION TITLE
PrimeVue with PurgeCSS

### DISCUSSION THREAD

**Discussion by yureckey** [primefaces/community#56] [PrimeVue]
I'm following along recommendations from docs how to start PrimeVue project with Vite. First minor hiccup is that in the docs there is no mentioning that you need to install Primeflex (while code samples from docs using it). But my question is - ok, I've added Primeflex and build minimal project with only button component - after build, the project is 500kb css + 150kb js - looks a bit much, can it be reduced in size? In the primeflex docs it is mentioned to use purgecss, but there is no reference config.

**Comment by atkntepe** [comment]
Hi,  for the file sizes, we're actively working on Primeflex 4.0, which will support PostCSS. This will allow for more customization and significantly reduce the size. We aim to release this update as soon as possible.

In the meantime, you might want to explore our PrimeVue unstyled and Tailwind theme. I hope this helps.

**Reply by devAnky** [reply]
Is there an example config now? I cannot find one.

**Comment by cagataycivici** [comment]
PrimeFlex is optional though, which part of the docs say it? I think DataGrid uses it which we plan to break as a dependency.

**Reply by yureckey** [reply]
which part of the docs mentioning PurgeCSS? the one within PrimeFlex https://primeflex.org/installation#productionsize
